SConstruct

```diff
@@ -1021,43 +1021,47 @@ env['python_cmd_esc'] = quoted(env['python_cmd'])
 
 # Python 2 Package Settings
 cython_min_version = LooseVersion('0.23')
+numpy_min_version = LooseVersion('1.6.1')
```

Comment:
We could add a few more jobs to check older versions of NumPy. I don't really want to set a minimum version without testing it against that version regularly, though. Frankly, 1.4 is seven versions (and seven years) old, and if you're absolutely limited to that version of NumPy, you should update your code. I'd prefer to treat this like Python 3.2 - if it works with <1.6.1, awesome. If not, we're not going to guarantee that. I'll bump `compiling.rst` as part of #419.

Going forward, if Travis bumps NumPy (e.g., if they ever go to 14.04), we could install older versions with pip to test with, depending on what we want the minimum version to be. I think that since they froze the C-API at 1.7 (that's what those warning messages during compile mean, right?), that would be a decent minimum version once Travis updates (or if we feel like getting pip to work now).